output/unallocated.txt

```diff
@@ -0,0 +1,3 @@
+Loading all unallocated people...
```

Comment:
**Code Quality:** Always _gitignore_ the data files if they are not needed for running the app.